api/jobs.yaml

```diff
@@ -244,6 +280,46 @@ definitions:
       labels:
         type: object
         description: All job labels, after the update has been applied.
+  AggregationResponse:
+    description: >
+      Returns aggregated jobs with specified time range or label.
```

Comment:
nit: Something along the lines of:
"Jobs aggregated by label and status within the specified time frame"